frontend/src/metabase/css/dashboard.css

```diff
@@ -83,6 +83,25 @@
   fill: currentColor;
 }
 
+.Dashboard.Dashboard--night .bg-light {
+  background-color: var(--color-bg-black);
+}
+
+.Dashboard.Dashboard--night .bg-medium {
+  background-color: #596269;
```

Comment:
Do we have any similar value stored in css vars?

Reply:
I don't believe so. I had initially tried one of our vars but nothing was suitable. I don't know that this needs to truly be a var if it's only used here.

Reply:
True. If we end up needing it elsewhere in the future, we can make a new var then.